src/Parser/DemonHunter/Havoc/Modules/Abilities.js

```diff
@@ -7,11 +7,9 @@ import Haste from 'Parser/Core/Modules/Haste';
 
 import CoreAbilities from 'Parser/Core/Modules/Abilities';
 
-import UnleashedDemons from './Traits/UnleashedDemons';
 
 class Abilities extends CoreAbilities {
```

Comment:
I think you're still missing a bunch of spells from this, please check the console for errors/warnings and fix them.

Reply:
Will fix asap.